pkgs/development/guile-modules/guile-reader/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl, pkgconfig
+, lightning
+, gperf, guile, guile-lib, libffi }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  name = "guile-reader-${version}";
+  version = "0.6.2";
+
+  src = fetchurl {
+    url = "http://download.savannah.nongnu.org/releases/guile-reader/${name}.tar.gz";
+    sha256 = "0592s2s8ampqmqwilc4fvcild6rb9gy79di6vxv5kcdmv23abkgx";
+  };
+
+  patches = [(
+     fetchurl {
```

Comment:
This formatting might be a little more regular (2 space indentation and parentheses):

https://github.com/NixOS/nixpkgs/blob/19ef534fbb044f2b95e0043c6e0f8b582d94952b/pkgs/desktops/gnome-3/core/gnome-shell/default.nix#L54-L59